packages/select/src/MenuButton/MenuButton.tsx

```diff
@@ -80,9 +76,9 @@ const MenuButton = React.forwardRef<HTMLButtonElement, MenuButtonProps>(
           menuButtonFocusStyle[theme],
           {
             [menuButtonDeselectedStyles[theme]]: deselected,
+            [menuButtonErrorStyle[theme]]: state === State.Error,
```

Comment:
just a nit, but I noticed in Combobox you're creating a new var (`const isErrorState = state === State.error`). Should we strive for consistency there? (personally, I don't think it matters, just curious what others thought)